packages/rspack/rspack.config.js

```diff
@@ -21,6 +24,15 @@ module.exports = {
           fullySpecified: false,
         },
       },
+      {
```

Comment:
there's a bug when rspack import json, so use json-loader to workaround it temporarily